sled-agent/src/services.rs

```diff
@@ -3817,26 +3830,76 @@ impl ServiceManager {
             }
         };
 
-        info!(self.inner.log, "Setting up uplinkd service");
-        let smfh = SmfHelper::new(&zone, &SwitchService::Uplink);
+        info!(self.inner.log, "ensuring scrimlet uplinks");
+        let usmfh = SmfHelper::new(&zone, &SwitchService::Uplink);
```

Comment:
We have two different SMF helpers, as we are updating the properties for two different services in the loop below.